modules/multi-col-index-test/src/lib.rs

```diff
@@ -0,0 +1,61 @@
+use spacetimedb::{query, spacetimedb};
+
+#[spacetimedb(table)]
+#[spacetimedb(index(btree, name = "chunk_index", chunk_index))]
+#[spacetimedb(index(btree, name = "coordinate_index", x, z, dimension))]
+#[derive(Clone)]
+pub struct LocationState {
+    #[primarykey]
+    pub entity_id: u64,
+
+    pub chunk_index: u64,
+
+    pub x: i32,
+    pub z: i32,
+    pub dimension: u32,
+}
+
+struct OffsetCoordinatesSmall {
+    x: i32,
+    z: i32,
+    dimension: u32,
+}
+
+#[spacetimedb(init)]
+pub fn init() {
+    // Insert 10^6 rows.
+    let mut entity_id = 0;
+    for x in 0..10 {
+        for z in 0..10 {
+            for dimension in 0..10 {
+                if let Err(err) = LocationState::insert(LocationState {
+                    entity_id,
+                    chunk_index: 0,
+                    x,
+                    z,
+                    dimension,
+                }) {
+                    log::info!("LocationState::insert error: {err:?}");
+                }
+                entity_id += 1;
+            }
+        }
+    }
+}
+
+#[spacetimedb(reducer)]
+pub fn test() -> anyhow::Result<()> {
+    log::info!("BEGIN");
+
+    // Try to find a row somewhere in the middle.
+    let offset = OffsetCoordinatesSmall {
+        x: 80,
+        z: 80,
+        dimension: 80,
+    };
+    let q = query!(|ls: LocationState| ls.x == offset.x && ls.z == offset.z && ls.dimension == offset.dimension);
```

Comment:
What are we testing?

Reply:
I'd rather we don't add it if it's not being exercised by CI.

Reply:
Can we not just add a timer here to make sure it takes the appropriate amount of time based on the expected query plan? I've been asking for that for a while, and no one has explained why that cannot be done.